src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1031,6 +729,8 @@ def _clean_plant_parts(self, plant_parts_eia):
                 ["record_id_eia", "appro_record_id_eia"],
             )
             # we'll eventually take this out... once Issue #20
+            # I think this is no longer needed? with the new labeling
+            # of true grans there aren't any duplicate record_id_eia
             .drop_duplicates(subset=["record_id_eia"])
```

Comment:
This `_clean_plant_parts` function is called at the end of `MakePlantParts.execute`. I think this `drop_duplicates` is no longer needed since there are no duplicate `record_id_eia` with the new labeling of true granularities. Should `record_id_eia` still be set as the index?